server/handlers/queue.go

```diff
@@ -84,7 +87,32 @@ func Publish(c *gin.Context) {
 		return
 	}
 
-	job := engine.NewJob(namespace, queue, body, uint32(ttlSecond), uint32(delaySecond), uint16(tries), "")
+	attributes := make(map[string]string)
+	for key, vals := range c.Request.Header {
```

Comment:
check prefix should be case insensitive and we can move this part into a single function? like `parseAttributes` or others